merkle/smt/nodes.go

```diff
@@ -31,16 +31,14 @@ type Node struct {
 // Prepare sorts the nodes slice for it to be usable by HStar3 algorithm and
 // the sparse Merkle tree Writer. It also verifies that the nodes are placed at
 // the required depth, and there are no duplicate IDs.
-//
-// TODO(pavelkalinnikov): Make this algorithm independent of Node type.
```

Comment:
The background for adding this TODO was that I wanted to sort/check length of IDs in `Get` requests just like in `Set`. I removed it because I thought it's not worth it.

Shouldn't be in this PR really, so changed back to as it was.